pom.xml

```diff
@@ -62,7 +67,7 @@
 		<mysql-connector-j.version>8.0.32</mysql-connector-j.version>
 		<mariadb-java-client.version>3.1.2</mariadb-java-client.version>
 		<assertj-core.version>3.24.2</assertj-core.version>
-		<maven-plugin-testing-harness.version>3.3.0</maven-plugin-testing-harness.version>
+		<maven-plugin-testing-harness.version>4.0.0-alpha-1</maven-plugin-testing-harness.version>
```

Comment:
Any reason for using `Alpha` version considering alpha releases might be buggy?

Reply:
No, actually i will return